app/javascript/stylesheets/home-page/_headers-style/home-page_headers-style.scss

```diff
@@ -0,0 +1,4 @@
+.home-page_headers-style_white-bold {
```

Comment:
I don't see the point in this division. Moreover, due to nesting, it looks like an element that cannot be reused.